payload-test.json

```diff
@@ -0,0 +1,146 @@
+            "properties": {
+                "hbase.rootdir": "${hbase_root_path}",
+                "hbase.client.keyvalue.maxsize": "750000000",
+                "hbase.server.keyvalue.maxsize": "750000000",
+                "hbase.bulkload.retries.retryOnIOException": "true",
+                "hbase.bucketcache.bucket.sizes": "5120,9216,17408,33792,41984,50176,58368,66560,99328,132096,197632,263168,394240,525312,656384,787456,918528,1049600,1180672,1311744,1442816,1704960,1967104,2229248",
+                "hbase.client.write.buffer": "8388608",
+                "hbase.regionserver.handler.count": "4"
+            }
+        },
+        {
+            "classification": "hbase",
+            "properties": {
+                "hbase.emr.storageMode": "s3",
+                "hbase.emr.readreplica.enabled": "true"
+            }
+        },
+        {
+            "classification": "emrfs-site",
+            "properties": {
+                "fs.s3.consistent.retryPeriodSeconds": "3",
+                "fs.s3.consistent.retryCount": "5",
+                "fs.s3.consistent": "true",
+                "fs.s3.consistent.metadata.tableName": "DataGen-metadata",
+                "fs.s3.cse.materialsDescription.enabled": "true"
+            }
+        }
+    ],
+    "steps": [
+        {
+            "name": "debug",
+            "actionOnFailure": "TERMINATE_CLUSTER",
+            "jarPath": "command-runner.jar"
+        }
+    ],
+    "applications": [
+        "HBase",
+        "Hive",
+        "Spark"
+    ],
+    "instanceGroups": [
+        {
+            "name": "small_cluster_master",
+            "instanceRole": "MASTER",
+            "instanceType": "m5.xlarge",
+            "instanceCount": 1,
+            "ebsConfiguration": {
+                "ebsBlockDeviceConfigs": [
+                    {
+                        "volumeSpecification": {
+                            "volumeType": "gp2",
+                            "sizeInGB": 128
+                        },
+                        "volumesPerInstance": 1
+                    }
+                ],
+                "ebsOptimized": false
+            }
+        },
+        {
+            "name": "small_cluster_core",
+            "instanceRole": "CORE",
+            "instanceType": "m5.xlarge",
+            "instanceCount": 1,
+            "ebsConfiguration": {
+                "ebsBlockDeviceConfigs": [
+                    {
+                        "volumeSpecification": {
+                            "volumeType": "gp2",
+                            "sizeInGB": 128
+                        },
+                        "volumesPerInstance": 1
+                    }
+                ],
+                "ebsOptimized": false
+            }
+        }
+    ],
+    "terminationProtected": false,
+    "emrManagedMasterSecurityGroup": "TODO",
+    "emrManagedSlaveSecurityGroup": "TODO",
+    "serviceAccessSecurityGroup": "TODO",
+    "additionalMasterSecurityGroups": [
+        "TODO"
```

Comment:
This should be the common sec group, assuming one was created in previous tickets otherwise it can be an empty list for now.